diff --git a/node_modules/react-native-animated-stopwatch-timer/src/StopwatchTimer.tsx b/node_modules/react-native-animated-stopwatch-timer/src/StopwatchTimer.tsx
index 8232ba9..78473c1 100644
+++ b/node_modules/react-native-animated-stopwatch-timer/src/StopwatchTimer.tsx
@@ -8,7 +8,7 @@ import Animated, {
   withTiming,
 } from 'react-native-reanimated';
 import type { StyleProp, TextStyle, ViewStyle } from 'react-native';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View, Platform} from 'react-native';
 import useTimer from './useTimer';
 
 const DEFAULT_ANIMATION_DELAY = 0;
@@ -284,7 +284,7 @@ function Stopwatch(
 
 const styles = StyleSheet.create({
   container: {
-    flexDirection: 'row',
+    flexDirection: 'row-reverse',
     alignItems: 'center',
     overflow: 'hidden',
   },
